### PR TITLE
Add validation method

### PR DESCRIPTION
**Pull request by JihongJeong**
Train & validation loss will printed at each of 10 batches also for each epochs
If you don't want to use checkpoints you should change 'use_checkpoint' into 'False' (default is 'True') at line:20 of train.py

### CODE REVIEW

**Comment by justin4ai** [comment]
You got this. Thanks for your work! I'll give you back after reviewing. Note we need to start working on making presentation slides - please refer to https://www.notion.so/Presentation-32be749c85e441ad81a33ca8711db723?pvs=4

**Review by justin4ai — CHANGES_REQUESTED** [review]
Your update is perfect. Thanks a lot!

I have two things to say:
A) Since the model does have the memory for the previous epochs, you have to split validation set **outside the epochs** such that the validation set doesn't get changed in each epoch - the model might have already been trained on it in previous epochs

B) We'd better off using argparser for better code maintenance.

I apply B) and pushed it to ```detector-tmp```. I won't send a PR to ```detector``` branch for now. So please follow the steps:

1) Mount on ```detector-tmp``` branch
2) ```git pull``` such that you apply my update about using argparser
3) Please work on A) then push it to ```detector-tmp```.
4) Send me a new PR!

**Comment by justin4ai** [comment]
Although you still can use this pr without creating new one via committing more to ```detector-tmp``` after ```git pull```, I won't do that.

So please include the update details of this PR when you send a new PR.
